packages/server/__tests__/api/agencies.test.js

```diff
@@ -36,20 +36,34 @@ describe('`/api/organizations/:organizationId/agencies` endpoint', () => {
         fetchOptions.staff.headers.cookie = await getSessionCookie('user2@nv.gov');
     });
 
-    context('GET organizations/:organizationId/agencies (list an agency and its subagencies)', () => {
+    context('GET organizations/:organizationId/agencies', () => {
+        it('lists all agencies within a tenant', async () => {
+            // Will default to user's own agency ID
+            const response = await fetchApi('/agencies', agencies.admin.own, fetchOptions.admin);
+            expect(response.statusText).to.equal('OK');
+            const json = await response.json();
+            expect(json.length).to.equal(4);
```

Comment:
I think it's worth asserting on what the specific agency IDs in the response are. Same applies to the other tests added in this file.